.github/workflows/macos-cmake-build.yml

```diff
@@ -7,42 +7,16 @@ on:
 jobs:
   build-and-test:
 
-    name: ${{ matrix.toolchain }}
-    runs-on: ${{ matrix.os }}
-
-    strategy:
-      matrix:
-        toolchain:
-          - macos-clang
-
-        configuration:
-          - Debug
-
-        include:
-          - toolchain: macos-clang
-            os: macos-latest
-            compiler: clang
+    runs-on: macos-latest
```

Comment:
![image](https://user-images.githubusercontent.com/2794213/120919446-48aa2f80-c6c2-11eb-89a0-1f6c679f6233.png)
(if you don't know who this is, take a look at metalocalypse)

Reply:
I think there is no need for specifying _toolchain_, _OS_ and _compiler_ because _macos-latest_ has these configurations by default, but I may be misled.